src/lib.rs

```diff
@@ -576,14 +576,14 @@ impl PyEnsureFuture {
     }
 }
 
-fn call_soon_threadsafe(
-    event_loop: &Bound<PyAny>,
+fn call_soon_threadsafe<'py>(
+    event_loop: &'py Bound<PyAny>,
```

Comment:
```suggestion
    event_loop: &Bound<'py, PyAny>,
```
You want to tie this to the Python lifetime of the `Bound`.

Reply:
Agreed on this one 👍